README.md

```diff
@@ -82,7 +82,7 @@ See ["Installing ReactOS"](https://www.reactos.org/wiki/Installing_ReactOS) Wiki
 
 ## Testing
 
-If you discover a bug in ReactOS search on JIRA first - it might be reported already. If not report the bug providing logs and as many information as possible. 
+If you discover a bug in ReactOS search on JIRA first - it might be reported already. If not report the bug providing logs and as many information as possible.
 
```

Comment:
The statement "..providing logs and as many information as possible." might be improved by changing it to "... providing logs and as much information as possible."